### PR TITLE
add tool to update imagestreams and templates, and do so

### PR DESCRIPTION
**Pull request by jim-minter**
```release-note
update openshift imagestreams and templates to 04/04/19
```
nb: many imagestreams now use registry.redhat.io, so more reliance on our secret for them to work
added the registry.redhat.io secret to the openshift namespace to enable pulling images
minor sync to artifacts script to make it quit early if we didn't get as far as having an admin kubeconfig
pkg/sync/v5/content_test.go is the import script
added test to ensure the sync pod doesn't spin on updating objects

### CODE REVIEW

**Comment by mjudeikis** [comment]
/retest
https://github.com/openshift/openshift-azure/issues/1400

File: hack/artifacts.sh [hunk 1]
```diff
@@ -6,6 +6,11 @@ if [[ -z "$ARTIFACT_DIR" ]]; then
     exit 0
 fi
 
+if [[ ! -e $PWD/_data/_out/admin.kubeconfig ]]; then
+    echo "admin.kubeconfig not found, exiting"
+    exit 1
```
Comment (asalkeld) [inline hack/artifacts.sh]: are you sure you want to exit 1 instead of 0 like above?

**Comment by jim-minter** [comment]
previously passed tests, one bit change, merging

**Pull request merged**